src/libnet_raw.c

```diff
@@ -53,13 +53,13 @@ libnet_open_raw6(libnet_t *l)
 }
 
 int
-libnet_close_raw4(libnet_t *l)
+libnet_close_raw4(const libnet_t *l)
```

Comment:
Not sure if this possible to be `const`. After look inside `libnet_close_raw4` that `libnet_close_link_interface()` which called from[ libnet_close_raw4](https://github.com/libnet/libnet/blob/39034323a4a2a366973b0be8e676cc425cf2e453/src/libnet_raw.c#L56) has two declaration which are `libnet_close_link_interface(struct libnet_link_int *l)` look [here](https://github.com/libnet/libnet/blob/39034323a4a2a366973b0be8e676cc425cf2e453/src/libnet_link_nit.c#L96), [here](https://github.com/libnet/libnet/blob/39034323a4a2a366973b0be8e676cc425cf2e453/src/libnet_link_pf.c#L146), [here](https://github.com/libnet/libnet/blob/39034323a4a2a366973b0be8e676cc425cf2e453/src/libnet_link_snit.c#L122) and `libnet_close_link_interface(libnet_t *l)` look [here](https://github.com/libnet/libnet/blob/39034323a4a2a366973b0be8e676cc425cf2e453/src/libnet_link_win32.c#L191). If we follow the definition found in the header `libnet_close_raw4(libnet_t *l)` then all `libnet_close_link_interface` must be `libnet_close_link_interface(libnet_t *l)`. Or am i miss something?